Chinese/CrazyStone/CrazyStone.md

```diff
@@ -2,4 +2,4 @@
 他打算挖地洞进入藏宝地点，说起挖地洞，现在问题来了。
 他没学过挖洞的技能，况且手头也没有工具，独自一人徒手挖洞显然无法想象。正在他抓耳挠腮绞尽脑汁的紧要关头，上天灵光突然照亮了他的脑袋---要挖洞，找蓝翔！！！
 麦克二话不说，径直奔向重庆火车站。此时正值春运，麦克排了三天三夜，终于轮到自己却被告知开往山东的火车票都已售罄……麦克心灰意冷地走出火车站，却意外撞见了小贼中的一名，原来他也没买到去蓝翔的火车票。两人一阵唏嘘，
-相见恨晚，大排档喝到半夜。第二天早上6点麦克被广场舞震醒，发现自己一个人躺在床上。
\ No newline at end of file
+相见恨晚，大排档喝到半夜。第二天早上6点麦克被广场舞震醒，发现自己一个人躺在床上。迈克起来伸了个懒腰，正要好好欣赏一下跳广场舞大妈们的优雅身姿，听见有人叫自己的名字，原来是昨天碰到的小贼，手里拿着两套夹饼果子。麦克毫不客气的结果其中一套，与小贼一起大快朵颐。
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.